rector_examples/datetime_datetime_storage_format.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * This demonstrates the deprecated static calls that might be called from procedural code like `.module` files.
+ */
+
+/**
+ * A simple example.
+ */
+function simple_example() {
+  $format = DATETIME_DATETIME_STORAGE_FORMAT;
+
+}
+
+/**
+ * An example using the constant as an argument.
+ */
+function as_an_argument() {
+  $date = new DrupalDateTime('now', new \DateTimezone('America/Los_Angeles'));
```

Comment:
Do we need this line?

Reply:
@dww, Since this is a deprecation for a constant, assigning it to a variable seems fine as a minimum test.

If you want to illustrate a real use case, that's always great. We don't have hard rules, but we would like the examples to provide some context if it's helpful.

Do we need a `use` statement or a fully qualified namespace `Drupal\Core\Datetime\DrupalDateTime` so this is valid code?

Rector does try to resolve classes.

Reply:
Yup, it already has the use.